meta/src/html.rs

```diff
@@ -76,10 +90,19 @@ pub fn Html(
                     _ = el.set_attribute("dir", &value);
                 });
             }
+
+            if let Some(class) = class {
+                let el = document().document_element().expect("there to be a <html> element");
```

Comment:
Instead of this, you should add a `let el = el.clone();` at line 88 before it moves `el` into the effect for `dir`, just as the `lang` attribute clones at line 80 before it moves into its effect.